### PR TITLE
Manual email data

### PR DESCRIPTION
**Pull request by geramirez**
Updated additional broken emails.
Part of 18F/foia-hub/issues/335

### CODE REVIEW

File: contacts/data/U.S. DOL.yaml [hunk 1]
```diff
@@ -552,7 +552,7 @@ departments:
       simple_lowest_days: '1'
       simple_median_days: '2'
   service_center: 'Phone: (202) 693-5500'
-  top_level: false
+  top_level: true
```
Comment (konklone) [inline contacts/data/U.S. DOL.yaml:555]: Why did this change?

**Comment by konklone** [comment]
This looks great, thanks for collecting this @ramirezg. Just looking for clarity on the note I raised.

File: contacts/data/U.S. DOL.yaml (hunk 1 above)
Comment (geramirez) [inline contacts/data/U.S. DOL.yaml:555]: Just checked over the log. Jobs corps had always been intended to be top level, but the name was misspelled. The manual data was corrected in a previous commit, but not applied yet.

**Comment by konklone** [comment]
Thanks for investigating. Looks good to me!

**Pull request merged**